### ISSUE TITLE
Add prettier to development flow

### ISSUE THREAD

**Issue by josecolella** [stryker-mutator/stryker#1271]
### Summary

Hello @nicojs , I would like to add prettier to the development flow to ensure that all files are formatted correctly.

Please let me know any specifics before starting on this task, such as single quotes and maximum line length

**Comment by nicojs** [comment]
Awesome! Thanks for opening this issue and your offer to help. It is truly appreciated. 

How will the workflow work? The way I see it, it could work 3 ways

1, Run as a "prebuild" (or other)
1. Run as a githook every time you commit (prettier shouldn't change functionality, only styling)
1. Configure it in the editor (and document it very well).

We could also combine more of them. Editor support seems like a logical one.

As for your specifics:

* Single quotes
* Column size. How about 130? How does this work? Will long strings be broken up for example?

**Comment by simondel** [comment]
I would also prefer editor support :)

**Comment by nicojs** [comment]
I agree, but just editor support is not enough IMHO. I want a way to validate it as well. Don't know exactly what is possible and preferred though.

**Comment by kmdrGroch** [comment]
Since we are using TSLint, is it nessesary to use prettier?
If not, we can close this issue :)

**Comment by nicojs** [comment]
Yeah, actually, I'm looking to migrate away from tslint (since it [will be deprecated soon anyway](https://github.com/palantir/tslint/issues/4534)). 

Prettier [easily integrates with linters](https://prettier.io/docs/en/integrating-with-linters.html) especially eslint. So it would still be nice to have it.

A linter and a formatter are similar, but they still do different things.

(it's just a matter of time before someone creates a "prettier-eslint" do-it-all-tool that does both ... 😑 )
